sql/src/parser.rs

```diff
@@ -352,8 +352,39 @@ impl<'a> Parser<'a> {
                 );
             }
         }
+        let mut cols: Vec<ColumnDef> = vec![];
+        for mut column in columns {
+            let options = column.options;
+            let mut new_opts = vec![];
+            for option in options {
+                match &option.option {
+                    ColumnOption::DialectSpecific(ts) => {
```

Comment:
`ts` is confusing because `TS_KEY` is already introduced and maybe we can call it `tokens`.